src/content/docs/developer/modules/other/external-qa-checks.mdx

```diff
@@ -0,0 +1,246 @@
+
+## Request to the App from Crowdin for runQaCheckUrl
+
+Request payload example:
+
+```json collapse={25-66}
+{
+  "data": {
+    "translations": [
+      {
+        "id": 12345,
+        "stringId": 1234567,
+        "languageId": "fr",
+        "userId": 1,
+        "text": "La mise à jour est installé avec succès.",
+        "provider": null,
+        "pluralCategoryName": null,
+        "isPreTranslated": false,
+        "rating": 0
+      }
+    ],
+    "strings": [
+      {
+        "id": 1234567,
+        "key": "update_success",
+        "context": "Confirmation of successful software update",
+        "maxLength": null,
+        "text": "The update was successfully installed.",
+        "fields": []
+      }
+    ],
+    "project": {
+      "id": 123,
+      "type": 0,
+      "sourceLanguage": {
+        "id": "en",
+        "name": "English",
+        "twoLettersCode": "en",
+        "threeLettersCode": "eng",
+        "locale": "en-US",
+        "pluralCategoryNames": ["one", "other"],
+        "pluralRules": "(n != 1)",
+        "pluralExamples": [1, 2],
+        "textDirection": "ltr",
+        "dialectOf": null
+      },
+      "targetLanguages": [
+        {
+          "id": "fr",
+          "name": "French",
+          "twoLettersCode": "fr",
+          "threeLettersCode": "fra",
+          "locale": "fr-FR",
+          "pluralCategoryNames": ["one", "other"],
+          "pluralRules": "(n > 1)",
+          "pluralExamples": [1, 2],
+          "textDirection": "ltr",
+          "dialectOf": null
+        }
+      ],
+      "name": "Project Name",
+      "description": "Project Description",
+      "fields": []
+    },
+    "file": {
+      "id": 123,
+      "name": "filename.csv",
+      "title": null,
+      "context": null,
+      "type": "csv",
+      "path": "/filename.csv",
+      "fields": []
+    }
+  }
+}
+```
+
+## Expected Response from the App (Without QA issues)
+
+Response payload example:
+
+```json
+{
+  "data": {
+    "validations": [
+      {
+        "translationId": 123,
+        "passed": true
+      }
+    ]
+  }
+}
+```
+
+## Expected Response from the App (With QA issues)
+
+Response payload example:
+
+```json
+{
+  "data": {
+    "validations": [
+      {
+        "translationId": 456,
+        "passed": false,
+        "error": {
+          "message": "Example error message",
+          "suggestedFixes": [ // optional; only applicable if running the QA check synchronously
```

Comment:
Instead of a comment, use the [Adding labels to line markers](https://expressive-code.com/key-features/text-markers/#adding-labels-to-line-markers) feature (**Adding long labels on their own lines**)

![Screenshot 2024-10-09 at 14 41 40](https://github.com/user-attachments/assets/1af03f3c-03e7-475b-8054-3b4157fe2463)

Reply:
Updated this using the mentioned feature